deposit-transactions.md

```diff
@@ -0,0 +1,165 @@
+signature information.
+
+[EIP-155]: https://eips.ethereum.org/EIPS/eip-155
+
+Although in practice we define only one new Transaction Type we can distinguish between two distinct
+transactions which occur in the deposit block, based on their positioning. The first transaction
+MUST be the [L1 Attributes Deposit Transaction][l1-attributes-deposit-transaction], followed by a
+dynamic array of [Deposited Transactions][deposited-transactions] submitted to the Deposit Feed
+contract by accounts on L1.
+
+## L1 Attributes Deposit Transaction
+
+[l1-attributes-deposit-transaction]: #l1-attributes-deposit-transaction
+
+This transaction is a call to the [Layer 1 Attributes Predeploy][l1-attributes-predeploy] contract.
+
+This transaction MUST have the following values:
+
+1. `from` is the Depositor Account `0xdeaddeaddeaddeaddeaddeaddeaddeaddead0001`.
+1. `to` is `0x4200000000000000000000000000000000000014` (the address of the L1 attributes predeploy
+   contract).
+1. `value` is `0`
+1. `data` is an abi encoded call to the [L1 Attributes Predeploy] contract's `setL1BlockValues()`
+   function with correct values associated with the corresponding L1 block.
+
+## Special Accounts on L2
+
+The L1 Attributes Deposit Transaction involves two special purpose accounts:
+
+1. The L1 Attributes Depositor Account
+2. The L1 Attributes Predeploy
+
+### L1 Attributes Depositor Account
+
+[l1-attributes-depositor-account]: #l1-attributes-depositor-account
+
+The Depositor Account is an EOA with no known private key. It has the address
+`0xdeaddeaddeaddeaddeaddeaddeaddeaddead0001`. Its value returned by the `CALLER` and `ORIGIN`
+opcodes during execution of the L1 Attributes Deposit Transaction.
+
+### L1 Attributes Predeploy
+
+[l1-attributes-predeploy]: #l1-attributes-predeploy
+
+A predeployed contract on L2 at address `0x4200000000000000000000000000000000000014`, which holds
+certain block variables from the corresponding L1 block in storage, so that they may be accessed
+during the execution of the subsequent deposited transactions.
+
+The contract implements an authorization scheme, such that it only accepts state-changing calls from
+the [Depositor Account].
+
+The contract has the following solidity interface, and can be interacted with according to the
+[contract ABI specification][ABI].
+
+[ABI]: https://docs.soliditylang.org/en/v0.8.10/abi-spec.html
+
+```solidity
+interface L1BlockValues {
+
+  function setL1BlockValues(
+    uint256 number,
+    uint256 timestamp,
+    uint256 baseFee,
+    bytes32 hash
+  ) external;
+
+  function l1Number() view;
+  function l1Timestamp() view;
+  function l1BaseFee() view;
+  function l1Hash() view;
+}
+```
+
+## Deposited Transactions
+
+[deposited-transactions]: #deposited-transactions
+
+"Deposited Transactions" are derived from logs emitted by the [Deposit Feed
+contract][deposit-feed-contract] on L1.
+
+### Deposit Feed Contract
```

Comment:
What about calling it "Deposit Contract" - I feel like as though no info would be lost?

Deposits don't really behave like a feed, because they get "chunked" into L1 blocks for the purpose of assigning them to an epoch (at a later specification stage).